test/BUILD

```diff
@@ -157,6 +164,13 @@ scala_library(
            glob(["src/main/scala/scala/test/mix_java_scala/*.java"]),
     jvm_flags = ["-Xms1G", "-Xmx4G"],
 )
+#Mix java scala
+scala_library(
+    name = "MixJavaScalaLibWithJvmFlags",
```

Comment:
ok, why do we need `OtherLibButWithJvmFlags`?
Can we also remove the jvm_flags from `MixJavaScalaLib` and only retain it in the new target?
Last question what is protected by `MixJavaScalaLibWithJvmFlags` (assuming `MixJavaScalaLib` doesn't have jvm_flags)? How is it protected? What test will break?